# set correct uid/gid for mysql, from OE (8836cd807198c224ed23b05fa4f708f096881b27).
# however poky doesn't include mysql so far. Remove it from patch list, but keep
# this patch for a while
#
# comment added by Kevin Tian <kevin.tian@intel.com>, 2010-07-06
--- base-passwd/group.master.orig	2005-07-08 06:36:07.717990112 +0200
+++ base-passwd/group.master	2005-07-08 06:36:32.000000000 +0200
@@ -36,4 +36,5 @@
 staff:*:50:
 games:*:60:
 users:*:100:
+mysql:*:64001:
 nogroup:*:65534:
